docs-ref-services/site-recovery.md

```diff
@@ -21,15 +21,20 @@ Site Recovery allows you to automate replication of Azure VMs between regions, o
 
 Learn more about [Azure Site Recovery](https://docs.microsoft.com/en-us/azure/site-recovery/site-recovery-overview).
 
-## Management Package
+## Management package
 
 ### Install npm module
+
+Install the Azure recovery servics npm module

```

Comment:
Should we capitalize all product names? Should this be `Install the Azure Site Recovery npm package`?